scripts/cfos/make_dataset_data_tables.py

```diff
@@ -129,7 +134,10 @@ def load_excel_input(excel, allowed_dataset_cols, allowed_dataset_tables, skipro
     raw_dataset_dict = pd.read_excel(excel, sheet_name=None, skiprows=skiprows, index_col=None, usecols=lambda x: x in allowed_dataset_cols)
     processed_dataset_dict = {k:v for (k,v) in raw_dataset_dict.items() if k in allowed_dataset_tables}
     print("Success: Excel file has been loaded into a dataframe. Note that any columns that are not defined in the schema will be ignored.")
-    print(processed_dataset_dict)
+    for key, value in processed_dataset_dict.items():
+        print("table: " + key)
+        print(key + " columns: " + str(value))
+        print("\n")
```

Comment:
I'd also put this newline outside of the for-loop so you don't have a break between every column name, but rather between every table. but that's just personal preference - if you intended to have a space between every column name, feel free to ignore me

Reply:
d'oh you're right! thanks!

Reply:
Actually this for loop needed to be changed in a slightly different way. The key is the table name, the value is the dataframe itself which has the columns. But I wasn't grabbing the columns correctly and it was getting wonky.